uint8 gear   
uint16 speed
uint16 steer
uint16 brake
uint32 encoder
uint16[] garmin

